workshop/exercise-3/README.md

```diff
@@ -1,57 +1,69 @@
-The knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
+The Knative client, `kn`, aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
 
 # Our First Knative Service
-Knative Serving supports deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
+Knative Serving enables rapid deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-The Knative Service object automatically manages the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new revision for each update of the service.
+The Knative Service object automatically manages the whole lifecycle for your workload. A Service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new revision for each update of the service.
 
 ![](https://github.com/knative/serving/raw/master/docs/spec/images/object_model.png)
 
 # Deploy our Application to Knative using kn
 We've already created an image on dockerhub that contains the first version of our Fibonacci application. If we call the `/` endpoint, and pass in a `number` parameter, we should get the first `n` numbers of the Fibonacci sequence.
 
-1. Deploy the application:
+1. Deploy the application. We will create a Knative Service named fib-knative which will run our fib-knative image on dockerhub. We'll give this service the revision-name `fib-knative-one`. If no revision name is defined, the system will automatically generate a revision name.
 
     ```
-    kn service create --image docker.io/ibmcom/fib-knative fib-knative
+    kn service create fib-knative --image docker.io/ibmcom/fib-knative --revision-name fib-knative-one
```

Comment:
I wonder if there's a way to make the first cmd they issue not have the --revision-name flag. To a newbie it can be confusing and make them think they need to specify it all the time. Perhaps start w/o it but then when we go to deploy the 2nd version we could give the first version a nice name, then deploy v2.

Reply:
Then we can explain why human picked names are sometimes need - but in other times (if we're doing a 100% roll-out) we don't need them at all - the generated names are just fine.

Reply:
I agree with that, will update.